huxley/www/views.py

```diff
@@ -21,6 +22,7 @@ def index(request):
 
     context = {
         'user_json': json.dumps(user_dict).replace('</', '<\\/'),
+        'conference_session': Conference.objects.filter(session=64)[0].session,
```

Comment:
I think that is ultimately the plan but this is just there for a filler while @tdowds builds the conferenceStore